lib/evss/gi_bill_status/amendment.rb

```diff
@@ -4,6 +4,22 @@
 
 module EVSS
   module GiBillStatus
+    ##
+    # Model for an amendment made to a user's enrollment
+    #
+    # @!attribute on_campus_hours
+    #   @return [Float] The number of credit hours the user was on campus
+    # @!attribute online_hours
+    #   @return [Float] The number of credit hours the user took online
+    # @!attribute yellow_ribbon_amount
+    #   @return [Float] The institution's contributions to the Yellow Ribbon Program
```

Comment:
Just to be certain, this is financial contributions in cents, right?

Reply:
Great question; I'm actually not sure. Looking into it now